src/ecs/util/resources.rs

```diff
@@ -2,14 +2,17 @@
 
 use util::circular_buffer::CircularBuffer;
 
-/// The FPSCounter resource needed by the FPSCounterSystem
-/// ## Examples
-/// Add it to your resources to be able to use the FPSCounterSystem
-/// world.add_resource(FPSCounter::new(20));
+/// The FPSCounter resource needed by the FPSCounterSystem.
 ///
-/// Usage:
-/// Get the FPSCounter resource from the System or from the world.
+/// Add it to your resources with id 0 to be able to use the FPSCounterSystem.
+///
+/// # Usage:
+/// Get the FPSCounter resource from the world then call something like:
+///
+/// ```rust,no_run
```

Comment:
Please add the missing imports to this tests and remove the `no_run`; CI is currently failing because of this example I think.